README.md

```diff
@@ -1061,6 +1062,19 @@ When an expression falls outside the range, the function returns:
 {{ dbt_utils.width_bucket(expr, min_value, max_value, num_buckets) }}
 ```
 
+#### listagg ([source](macros/cross_db_utils/listagg.sql))
+This macro returns the concatenated input values from a group of rows separated by a specified deliminator. 
+
+**Args**:
+- `measure` (required): The expression (typically a column name) that determines the values to be concatenated. To only include distinct values add keyword DISTINCT to beginning of expression (example: 'DISTINCT column_to_agg').
+- `delimiter_text` (required): Text representing the delimiter to separate concatenated values by.
+- `order_by_clause` (optional): An expression (typically a column name) that determines the order of the concatenated values.
+- `limit_clause` (optional): Only for bigquery. Specifies the maximum number of values to be concatenated.
+
+**Usage:**
+```
+{{ dbt_utils.listagg(measure='column_to_agg', delimiter_text="','", order_by_clause="order by order_by_column", limit_clause="limit 10") }}
+```
```

Comment:
Right now, we don't (not necessarily for any good reason, in fairness) document the cross-db macros. I think it's because they're generally not intended for public consumption, so much as for usage by package maintainers themselves. 

Since they're moving to dbt core soon anyway (https://github.com/dbt-labs/dbt-core/issues/4813), I think we stick with that rule.

Reply:
Alright, I will remove this piece!